src/stories/Library/card-list-info-box/CardListInfoBox.tsx

```diff
@@ -0,0 +1,40 @@
+type CardListInfoBoxProps = {
+  title?: string;
+  html?: string;
+  buttonText?: string;
+};
+
+const CardListInfoBox = ({ title, html, buttonText }: CardListInfoBoxProps) => {
```

Comment:
I think `buttonLabel` would be a better name, also in the dpl-react project, to avoid confusion with translation logic naming that is postfixed with Text.